test/hash_recursive_variant_test.cpp

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2011-2014
+// Antony Polukhin
```

Comment:
Put you name in here, instead of mine :)
Also change the copyright date to 2016